src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/commands.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=line-too-long
+
+from azure.cli.core.commands import cli_command
+import azure.cli.command_modules.documentdb
```

Comment:
Shouldn't import from `azure.cli.command_modules.documentdb` as there's a big perf. hit.
Is this needed in order to load the commands?